src/Quadrature/quadrature.jl

```diff
@@ -154,14 +172,14 @@ end
 ######################
 
 """
-    FacetQuadratureRule{shape}([::Type{T}, [quad_rule_type::Symbol,] order::Int)
+    FacetQuadratureRule{shape}([::Type{T},] [quad_rule_type::Symbol,] order::Int)
     FacetQuadratureRule{shape}(face_rules::NTuple{<:Any, <:QuadratureRule{shape}})
     FacetQuadratureRule{shape}(face_rules::AbstractVector{<:QuadratureRule{shape}})
 
 Create a `FacetQuadratureRule` used for integration of the faces of the refshape `shape` (of
 type [`AbstractRefShape`](@ref)). `order` is the order of the quadrature rule.
-`quad_rule_type` is an optional argument determining the type of quadrature rule.
-Currently the :polyquad, `:legendre` and `:lobatto` rules are implemented, depending on the reference element.
+It defaults to the default rules for the individual face reference elements. Elements with mixed facet types
+do not have a way of specifying the quadrature rule via symbol for now.
```

Comment:
```suggestion
```suggestion
If not provided, the default `quad_rule_type` for each facet's reference shape is used.
For non-default `quad_rule_type`s on cells with mixed facet types (e.g. `Wedge`), the `face_rules` must be provided explicitly. 
```
```